packages-exp/functions-exp/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@firebase/functions-exp",
+  "version": "0.0.800",
+  "description": "",
+  "private": true,
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "main": "dist/index.node.cjs.js",
+  "browser": "dist/index.esm.js",
+  "module": "dist/index.esm.js",
+  "esm2017": "dist/index.esm2017.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "build": "rollup -c && yarn api-report",
+    "build:deps": "lerna run --scope @firebase/'{app-exp,functions-exp}' --include-dependencies build",
```

Comment:
Don't need to explicitly specify `app-exp` anymore because it has been added in devDependencies.